package.json

```diff
@@ -107,7 +107,7 @@
                 },
                 "bazel.queriesShareServer": {
                     "type": "boolean",
-                    "default": false,
+                    "default": true,
                     "description": "Use the same Bazel server for queries and builds. By default, vscode-bazel uses a separate server for queries so that they can be executed in parallel with builds. You can enable this setting if running multiple Bazel servers has a negative performance impact on your system, but you may experience degraded performance in Visual Studio Code for operations that require queries."
```

Comment:
Just one note, the description would have to be adapted because it assumes that default is false. Before I do that I would like to know the opinion of the contributors if they would agree on having default to true or not.

Reply:
@coeuvre @philwo any feedback?